frontend/src/Store/Actions/seriesActions.js

```diff
@@ -377,8 +379,56 @@ export const actionHandlers = handleThunks({
           });
         });
     }, MONITOR_TIMEOUT);
-  }
+  },
+
+  [UPDATE_SERIES_MONITOR]: function(getState, payload, dispatch) {
+    const {
+      id,
+      monitored,
+      monitor
+    } = payload;
+
+    const series = [];
+    const seriesToUpdate = { id };
+    if (payload.hasOwnProperty('monitored')) {
+      seriesToUpdate.monitored = monitored;
+    }
+    series.push(seriesToUpdate);
 
+    dispatch(set({
+      section,
+      isSaving: true
+    }));
+
+    const promise = createAjaxRequest({
+      url: '/seasonPass',
+      method: 'POST',
+      data: JSON.stringify({
+        series,
```

Comment:
No need to store the series to update as a variable then add it to an array, instead just do:

```
series: [
  seriesToUpdate
]
```

Reply:
@markus101 apologies I am completely new to JS, to be clear if I were to do this then I could get rid of both the if block and the push correct?

Reply:
answered this myself, yes you can - just working on the other fixes before I commit again

Reply:
Done